### PR TITLE
Integrations specs

### PR DESCRIPTION
**Pull request by Mwapsam**
### Blog app - integration specs for views

**In this project, I created integration tests for all of the views used in the project. I covered all the user stories (or user workflows) that I want the users to experience while taking into consideration the possible errors they may cause. to achieve all this I did the following:**

- _Used Capybara to write integration tests for each view in the project_
- _Tested all features in the views_

### CODE REVIEW

File: spec/integration/post_show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
```
Comment (SajjadAhmad14) [inline spec/integration/post_show_spec.rb:2]: For `User's Posts Index`:

Please add a test that asserts the user can see the user's profile picture.
Please add a test that asserts the user can see the user's username.
Please add a test that asserts the user can see the number of posts the user has written.
Please add a test that asserts the user can see a post's title.
Please add a test that asserts the user can see some of the post's body.
Please add a test that asserts the user can see the first comments on a post.
Please that asserts user can see how many comments a post has.
Please add a test that asserts the user can see how many likes a post has.
Please add a test that asserts users can see a section for pagination if there are more posts than fit on the view.
Please add a test that redirects to the post's show page when you click on a post.

There are some tests failing. Kindly make sure that all the tests pass.
fails1
fails2

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Mwape, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: spec/integration/post_show_spec.rb (hunk 1 above)
Comment (Mwapsam) [reply spec/integration/post_show_spec.rb:2]: Thanks for pointing out, I have fixed all of them now 😀

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🍾🎉**
Hello @Mwapsam 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**